skops/io/_general.py

```diff
@@ -108,6 +106,32 @@ def function_get_instance(obj, src):
     return loaded
 
 
+def partial_get_state(obj, dst):
+    _, _, (func, args, kwds, namespace) = obj.__reduce__()
+    res = {
+        "__class__": "partial",  # don't allow any subclass
+        "__module__": get_module(type(obj)),
+        "content": {
+            "func": _get_state(func, dst),
```

Comment:
So this isn't adding any security issues as far as I can tell, but thinking about it, why can't a user pass func to be `exec` and the arg to be arbitrary code to `function_get_state`?

Reply:
Yes, you're right, func could be eval or exec. Do you think we should check that?

Reply:
I was going to write an audit method separately and have `load` call it, or the user themselves to call it, there we can try to catch things.